dragonchain/lib/dao/smart_contract_dao.py

```diff
@@ -29,7 +29,7 @@
 
 def get_contract_by_txn_type(txn_type: str) -> "smart_contract_model.SmartContractModel":
     """Searches for a contract by txn_type"""
-    results = redisearch.search(index_name=FOLDER, query=txn_type)["results"]
+    results = redisearch.search(index="sc", query_str=f"@txn_type:{txn_type}")["results"]
```

Comment:
this wont' work. Search used to pull from S3. We'll need to do that manually now. Also I don't even think ['results'] is correct on a redisearch result